Makefile.am

```diff
@@ -217,6 +217,8 @@ uninstall-local::
 	find $(DESTDIR)$(debugdir)$(pkgdatadir) -type d -empty -delete
 dist-hook:: $(WEBPACK_INPUTS) $(WEBPACK_OUTPUTS) $(WEBPACK_PO)
 	tar -cf - $^ | tar -C $(distdir) -xf -
+	rm -f $(distdir)/dist/*/manifest.json
+	rm -f $(distdir)/pkg/*/manifest.json
```

Comment:
What would actually create pkg/*/manifest.json ?  I don't have any such files here, and that sounds wrong - they should only ever be created in dist/.

Reply:
They are created in `pkg` and then copied to `dist` by `./Makefile.am` with these rules:

```
.json.in.json:
	$(SUBST_RULE)

dist/%/manifest.json: pkg/%/manifest.json
	$(COPY_RULE)
```

`make` has a feature were it removes intermediate files that didn't exist before its invocation.  This is probably why you don't have an `pkg/*/manifest.json` files.